ui/app/models/cluster.js

```diff
@@ -46,14 +46,8 @@ export default DS.Model.extend({
   //otherwise the particular mode will have the relevant mode attr through replication-attributes
   mode: attr('string'),
   allReplicationDisabled: and('{dr,performance}.replicationDisabled'),
-
   anyReplicationEnabled: or('{dr,performance}.replicationEnabled'),
 
-  modeState: computed('dr.{mode,state}', 'performance.{mode,state}', 'replicationMode', function() {
```

Comment:
turns out we didn't need this property here!

Reply:
Interesting!

Reply:
Interesting!  Looks like `modeState` wasn't getting calculated in the cluster?  Or was it being calculated but not passed down properly where it was being consumed?